modules/installation-aws-regions-with-no-ami.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-government-region.adoc
+
+[id="installation-aws-regions-with-no-ami_{context}"]
+= AWS regions without a published {op-system} AMI
+
+You can deploy an {product-title} cluster to Amazon Web Services (AWS) regions
+without native support for a {op-system-first} AMI or the AWS SDK. If a
+published AMI is not available for an AWS region, you must upload a custom AMI
+prior to installing the cluster.
+
+[NOTE]
+====
+Deploying to an AWS Government region is a common use case where a region would
+require a custom {op-system} AMI.
+====
+
+A region without native support for an {op-system} AMI is not available to
+select from the terminal during cluster creation because it is not published.
+However, you can install to this region by configuring the custom AMI in the
+`install-config.yaml` file.
+
+The installation program imports the custom AMI to the user account
```

Comment:
Does this mean `importing AMI from other region (e.g. us-east-1) into current region` ? 
If yes, I do not think it works on US-Gov. Since AMI is provided by user manually (as described in section `Uploading a custom RHCOS AMI in AWS`), installer does not do `import` action.

Reply:
I gathered this from the tech enablement presentation for this topic. The presentation did use the word "copy" which is probably a better choice of words. I can make that update if this is still true.

@abhinavdahiya can you confirm the custom AMI is indeed copied to the cluster's user account by the installer?

Reply:
I think the installer will always try to copy the AMI from us-east-1 if it doesn't have one for that region. So for commercial regions this will work but for gov cloud this will not.

So I think as-is in this file saying the installer will copy the ami for the cluster is accurate. We just need to explicity state that for gov cloud people need to copy/import on their own.